src/wallet/rpcwallet.cpp

```diff
@@ -2652,8 +2652,8 @@ UniValue bumpfee(const JSONRPCRequest& request)
             "By default, the new fee will be calculated automatically using estimatefee.\n"
             "The user can specify a confirmation target for estimatefee.\n"
             "Alternatively, the user can specify totalFee, or use RPC setpaytxfee to set a higher fee rate.\n"
-            "At a minimum, the new fee rate must be high enough to pay a new relay fee (relay fee amount returned\n"
-            "by getnetworkinfo RPC) and to enter the node's mempool.\n"
```

Comment:
I think it'd be good to have this if only so this documentation can be less vague. If I'm trying to write an RPC call that passes a totalFee and the documentation just says I need to "pay a new relay fee", what am I supposed to do at the point to find out what value I need to pass for the call to complete reliably? Search for "bitcoin relay fee"? Look at the source code? I think I'd be better off with documentation that says I need to increase the fee by at least the amount returned in getnetworkinfo.